src/lib/block/View/SocialFeed.vue

```diff
@@ -1,14 +1,69 @@
 <template>
     <div>
-        {{ options }}
+      <div v-if="twitterHandle">
+        <Timeline :id="twitterHandle" sourceType="profile" :options="{ tweetLimit: '3' }">
+          <div class="spinner"></div>
+        </Timeline>
+      </div>
+      <div v-else>
+        <div v-if="socialNetwork === ''">
+          <p>No input for displaying social feed...</p>
+        </div>
+      </div>
     </div>
 </template>
 <script>
 import optionsPropMixin from './mixins/optionsProp'
+import Timeline from 'vue-tweet-embed/timeline'
+import { extractSocialUrl } from '../kind/SocialFeed'
+
+/// ///////////////////////////////////
+/// INFO: Custom class to Twitter timeline
+/// add :widget-class="`my-custom-class-1 my-custom-class-2`"
+/// to the <Timeline/> element
+/// ///////////////////////////////////
 
 export default {
   mixins: [
     optionsPropMixin,
   ],
+  components: {
+    Timeline,
+  },
+  data () {
+    return {
+      url: '',
+      twitterHandle: '',
+      socialNetwork: '',
+      socialFeed: null,
+    }
+  },
+  methods: {
+    extractSocialUrl (o) {
```

Comment:
I've come across in the past where strange things happen in calling methods directly as they are imported. Seems to work better when wrapped. Especially if you call them from the template. Since then it has just been habit to always wrap in the a local method first.